packages/nuget-exe/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nuget-exe",
+  "version": "4.9.1",
+  "description": "Repackaged NuGet.exe bianry",
```

Comment:
NuGet is under the apache license, so redistributing it should be kosher.

Reply:
Nit: typo in binary